pysgrid/sgrid.py

```diff
@@ -514,14 +525,15 @@ def save_as_netcdf(self, filepath):
             # populate variables with data
             grid_time[:] = self.grid_times[:]
             grid_center_lon[:] = self.centers[..., 0]
-            grid_center_lat[:] = self.centers[..., 1] 
-            grid_node_lon[:] = self.nodes[..., 0]
-            grid_node_lat[:] = self.nodes[..., 1]
+            grid_center_lat[:] = self.centers[..., 1]
+            if self.nodes is not None: 
+                grid_node_lon[:] = self.nodes[..., 0]
+                grid_node_lat[:] = self.nodes[..., 1]
```

Comment:
Consider using this syntax here and above (and possibly merging both codes into `SGridND` class making SGrid2D and SGrid3D simpler).  It is virtually the same code, right?  Just check for the dimensionality in `SGridND` and add a property there.

Does this make sense to you?  I haven't read all the code yet...